src/common/records.lua

```diff
@@ -53,7 +53,21 @@ end
 --- Get all records from the ANT
 ---@return string The encoded JSON representation of all records.
 function records.getRecords()
-	return json.encode(Records)
+	local antRecords = utils.deepCopy(Records)
+	assert(antRecords, "Failed to copy Records")
+	local recordEntries = utils.entries(antRecords)
+	-- sort the records alphabetically, ensuring "@" record is first
+	table.sort(recordEntries, function(a, b)
+		if a[1] == "@" then
+			return true
+		end
+		if b[1] == "@" then
+			return false
+		end
+		return a[1] < b[1]
+	end)
+
+	return json.encode(recordEntries)
```

Comment:
why do we need to encode here?

Reply:
[30824b5](https://github.com/ar-io/ar-io-ant-process/pull/35/commits/30824b5c9392d826e562c4d833fb7ebed4fe01a7)